taskfiles/k8s.yml

```diff
@@ -156,12 +156,23 @@ tasks:
         silent: true
       - 'echo "~~~ :docker: Build & Push operator container"'
       - docker buildx build
-        --platform linux/arm64,linux/amd64
+        --provenance false --sbom false --platform linux/amd64
         --file operator/Dockerfile
         --target manager
-        --tag docker.io/redpandadata/{{.OPERATOR_REPO}}:{{.TAG_NAME}}
+        --tag docker.io/redpandadata/{{.OPERATOR_REPO}}:{{.TAG_NAME}}-amd64
         --push
         ./dist
+      - docker buildx build
+        --provenance false --sbom false --platform linux/arm64
+        --file operator/Dockerfile
+        --target manager
+        --tag docker.io/redpandadata/{{.OPERATOR_REPO}}:{{.TAG_NAME}}-arm64
+        --push
```

Comment:
nit: do we want to push the intermediate tags? Seems like it would be cleaner to just have the multi-arch image.

Reply:
It would be cleaner, but I don't know how to perform such push without changes to the docker engine that runs in our buildkite runner.

Reply:
Wouldn't we just need to remove the `--push` argument or does that do something other than pushing the built image to dockerhub?